src/pks/transport/transport_ats_ti.cc

```diff
@@ -29,98 +29,105 @@ namespace Transport {
 //
 // This uses a first-order donor upwind scheme.
 //
-// DEV NOTE: this requires that tcc is ghosted and scattered, and that cq_flux is
+// DEV NOTE: this requires that tcc is ghosted and scattered, and that c_qty is
 // owned.
 void
-Transport_ATS::AddAdvection_FirstOrderUpwind_(double t_old,
-        double t_new,
-        const Epetra_MultiVector& tcc,
-        Epetra_MultiVector& cq_flux)
+Transport_ATS::AddAdvection_FirstOrderUpwind_(
+  double t_old, double t_new,  // old and new time
+  const Epetra_MultiVector& tcc,  // concentration in a cell at old time [mols C / mols H2O]
+  Epetra_MultiVector& c_qty,  // component quantity in a cell [mols C]
+  Epetra_MultiVector& cq_flux)  // component flux across a face [mols C / s]
 {
-  double dt = t_new - t_old;
+  double dt = t_new - t_old;  // time step [s]
 
+  // total number of faces in the domain
   int nfaces_all =
     mesh_->getNumEntities(AmanziMesh::Entity_kind::FACE, AmanziMesh::Parallel_kind::ALL);
+  
+    // get water flux (flux_key_) for each face at next time step (tag_next_)
```

Comment:
Do we want to rename flux_key_ --> water_flux_key_?  Or something similar to distinguish these?

Reply:
Good point. Let's use water_flux_key_ to distinguish it with other flux_keys_